<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NOTE</class>
 <widget class="QDialog" name="NOTE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_note">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NOTE</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_text">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EXIT">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>DATE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
